File: scalardb-sql-jdbc-sample/src/main/java/sample/command/RepaymentCommand.java
package sample.command;

import java.util.concurrent.Callable;
import picocli.CommandLine.Command;
import picocli.CommandLine.Parameters;
import sample.Sample;

@Command(name = "Repayment", description = "Repayment")
public class RepaymentCommand implements Callable<Integer> {

  @Parameters(index = "0", paramLabel = "CUSTOMER_ID", description = "customer ID")
  private int customerId;

  @Parameters(index = "1", paramLabel = "AMOUNT", description = "amount of the money for repayment")
  private int amount;

  @Override
  public Integer call() throws Exception {
    new Sample().repayment(customerId, amount);
    return 0;
  }
}

File: dotnet/microservice-transactions-sample-with-shared-cluster-with-linq/Client/Commands/RepaymentCommand.cs
using System.CommandLine;
using MicroserviceTransactionsSample.Client.Commands.Binders;
using MicroserviceTransactionsSample.Rpc;
using static MicroserviceTransactionsSample.Rpc.CustomerService;

namespace MicroserviceTransactionsSample.Client.Commands;

public static class RepaymentCommand
{
    private const string Name = "Repayment";
    private const string Description = "Make a repayment";

    private const string ArgName1 = "customer_id";
    private const string ArgDescription1 = "customer ID";
    private const string ArgName2 = "amount";
    private const string ArgDescription2 = "repayment amount";

    public static Command Create()
    {
        var customerIdArg = new Argument<int>(ArgName1, ArgDescription1);
        var amountArg = new Argument<int>(ArgName2, ArgDescription2);
        var repaymentCommand = new Command(Name, Description)
        {
            customerIdArg,
            amountArg
        };

        repaymentCommand.SetHandler(repay,
                                    customerIdArg,
                                    amountArg,
                                    new CustomerServiceClientBinder());
        return repaymentCommand;
    }

    private static async Task repay(int customerId, int amount, CustomerServiceClient client)
    {
        var request = new RepaymentRequest { CustomerId = customerId, Amount = amount };
        var response = await client.RepaymentAsync(request);

        Console.WriteLine(response);
    }
}
